.travis.yml

```diff
@@ -3,11 +3,14 @@ rust:
   - stable
   - 1.38.0  # lowest rust release against which we guarantee compatibility.
 cache: cargo
+env:
+  - CHRONO='--features "chrono"'
```

Comment:
this isn't working because you quotes `"chrono"`. you can test this yourself on a local shell by doing the same as travis-ci:
```sh
$ export CHRONO='--features "chrono"'
$ cargo build $CHRONO
error: Package `r2d2-oracle v0.2.0 (Z:\development\r2d2-oracle)` does not have these features: `"chrono"`
$ export CHRONO='--features chrono'
$ cargo build $CHRONO
    Finished dev [unoptimized + debuginfo] target(s) in 1.32s
```

Reply:
Quotes worked on my machine (win 10) and that's how it's specified [in the book](https://doc.rust-lang.org/cargo/reference/manifest.html#usage-in-end-products). Maybe linux is different in that